src/sysen2/xqueue.297

```diff
@@ -1062,11 +1073,14 @@ repeat 2,[push p,y			; save rest of date
 	jrst quecr			; and stick in a cr/lf
 
 ;Output the filenames into the queue file.
-quefil:	move b,device
+quefil:	
+	move b,device
+ifn mit,[
 	move a,machin
 	camn b,['DSK,,]
 	 movem a,device
 	movei b,device
+]
```

Comment:
Both open and close bracket should move one line up.  At entry to RFN"PFN, B should have the address to a file name quartet.  I fixed this in #2271